scripts/export_hf_datasets_to_jsonl.py

```
@@ -0,0 +1,105 @@
+
+
+def resolve_repo_and_splits(repos: list[str], *, trust_remote_code: bool = False) -> tuple[str, list[str]]:
+    last_error: Exception | None = None
+    kwargs: dict[str, Any] = {}
+    if trust_remote_code:
+        kwargs["trust_remote_code"] = True
+    for repo in repos:
+        try:
+            splits = get_dataset_split_names(repo, **kwargs)
+            return repo, list(splits)
+        except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)